scripts/figma-synchronise-tokens.sh

```diff
@@ -22,7 +22,7 @@ function generateTokens {
   reduce (.[$theme].shades | to_entries | .[]) as $sentiment
     ({}; . + {"\($sentiment.key)": (
       reduce($sentiment.value | to_entries | .[]) as $shade
-        ({}; . + { "\($shade.key)": $shade.value.value })) })
+        ({}; . + { "\($shade.key)": ($shade.value.value | ascii_upcase) })) })
```

Comment:
```suggestion
        ({}; . + { "\($shade.key)": ($shade.value.value | ascii_downcase) })) })
```

To reduce amount of changes in snaps ?

Reply:
Sure thing :)